server/app/models/grid_service.rb

```diff
@@ -147,7 +147,7 @@ def stack_exposed?
   end
 
   def deploy_pending?
```

Comment:
To fix this for 1.2.1 before #2221 is ready, probably better to just leave `deploying?` as-is, and have the `GridServiceHealthMonitorJob` call something like `deploy_outstanding?`.

However, it still needs to ignore all of the ancient deploys that were marked as `started_at` but never actually run.